tools/OwaspComplianceChecker/server/compliance/checker.ts

```
@@ -0,0 +1,464 @@
+        passed = repositoryData.files.some((file: any) => 
+          file.name.toUpperCase() === "CODE_OF_CONDUCT.MD");
+        break;
+      case "gc-9":
+        passed = repositoryData.files.some((file: any) => 
+          /ROADMAP|MILESTONES/i.test(file.name)) || 
+          repositoryData.has_projects || 
+          (repositoryData.readme && repositoryData.readme.toLowerCase().includes("roadmap"));
+        break;
+      case "gc-10":
+        // Check if there have been commits in the last 180 days
+        const lastActivityDate = repositoryData.updated_at || repositoryData.pushed_at;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)